src/components/Sidebars/Payback.js

```diff
@@ -169,25 +194,51 @@ const Payback = ({ cdpId, reset }) => {
             }
           />
         </Grid>
-        {userStartedWithoutAllowance && (
+        {userState.startedWithoutProxy && (
+          <Grid gridTemplateColumns="2fr 1fr" gridColumnGap="s">
+            {!userState.proxyAddress && !userState.proxyLoading && (
```

Comment:
Rather than copying the component three times here and within the `startedWithoutAllowance` block, it's a little cleaner to just generate the text inside the component. Something like:
```jsx
<Text.p t="body">
  {proxyAddress ? proxy_created : (proxyLoading ? creating_proxy : create_proxy)}
</Text.p>
```